binding/dotnet/README.md

```diff
@@ -24,6 +24,10 @@ Leopard is an on-device speech-to-text engine. Leopard is:
 
 ## Compatibility
 
+Platform compatible with .NET 6.0+:
```

Comment:
Can you move this to below .NET Core 3.1/RPi compatibility? The list should be in ascending order of limited support.